xds/internal/testutils/e2e/clientresources.go

```diff
@@ -207,7 +207,7 @@ func DefaultServerListener(host string, port uint32, secLevel SecurityLevel) *v3
 									RouteConfig: &v3routepb.RouteConfiguration{
 										Name: "routeName",
 										VirtualHosts: []*v3routepb.VirtualHost{{
-											Domains: []string{"lds.target.good:3333"},
+											Domains: []string{"127.0.0.1*", "my-service-fallback*", "*"},
```

Comment:
This `my-service-fallback*` and other domains are magic strings that a person reading a test has to mentally follow from where it is used to here. That puts a lot of cognitive load on the reader, more so here because the test and this function are in different packages.

One option would be to accept the list of domains as a parameter to `DefaultServerListener` like how we accept host, port and secLevel. You could just leave a TODO here and do it in a follow up PR (since it will probably touch other test files as well).

See: https://docs.google.com/document/d/1iMGZ6R2oDADAc0vYWgxVP5dyu3iELkcN2s0GYDfE6JI/edit

Reply:
Do you want me to just make this "*", which would match anything?

Reply:
With a comment, rather than a variable that callers pass in? Since this isn't testing routing, and that would be work for the caller to figure out how to match :authority to virtual host.

Reply:
Yeah I did that for both IPV4 and IPV6 filter chains. I think that makes the most sense logically, so the caller doesn't have to deal with it.